gnomad_constraint/resources/resource_utils.py

```diff
@@ -116,6 +123,30 @@ def get_training_dataset(
     )
 
 
+def get_models(
+    model_type: str,
+    version: str = CURRENT_VERSION,
+    genomic_region: str = "autosome_par",
+    test: bool = False,
+) -> str:
+    """
+    Return path to a pickle file that saves the expression of model.
```

Comment:
these aren't actually pickles, are they? this might be more accurate:
```suggestion
    Return path to a HailExpression that contains desired model type.
```